src/main/java/seedu/duke/userinterface/command/Help.java

```diff
@@ -1,6 +1,14 @@
 package seedu.duke.userinterface.command;
 
+import seedu.duke.userinterface.Mode;
+
 public class Help extends CliCommand {
+
+    public Help(String argument, Mode uiMode) {
+        this.setUiMode(uiMode);
```

Comment:
We should move this to a setter as discussed in the meeting right? Or am I remembering incorrectly?

Reply:
I'm not sure. What do others think?